protocols/migration/migration_reports_4_0_0_to_reports_3_0_0.py

```diff
@@ -91,3 +94,186 @@ def migrate_genomic_feature(self, old_genomic_feature):
             other_ids=old_genomic_feature.otherIds,
         )
         return self.validate_object(object_to_validate=new_genomic_feature, object_type=self.new_model.GenomicFeature)
+
+    def migrate_interpretation_request_rd(self, old_instance):
+        """
+        Migrates a reports_4_0_0.InterpretationRequestRD into a reports_3_0_0.InterpretationRequestRD
+        :type old_instance: reports_3_0_0.InterpretationRequestRD
+        :rtype: reports_6_0_0.InterpretationRequestRD
+        """
+        new_instance = self.convert_class(self.new_model.InterpretationRequestRD, old_instance)
+        new_instance.versionControl = self.new_model.VersionControl()
+        new_instance.InterpretationRequestID = old_instance.interpretationRequestId
+        new_instance.InterpretationRequestVersion = old_instance.interpretationRequestVersion
+        new_instance.TieringVersion = old_instance.tieringVersion
+        new_instance.analysisReturnURI = old_instance.analysisReturnUri
+        new_instance.TieredVariants = self.migrate_tiered_variants(old_tiered_variants=old_instance.tieredVariants)
```

Comment:
I think we could use the existing `self.migrate_reported_variants()`?

Reply:
good plan

Reply:
done